docs/deployment/configuration/general.rst

```diff
@@ -367,7 +96,7 @@ PodTemplate name will be used.
 
 
 Set the ``default-pod-template-name`` in FlytePropeller
---------------------------------------------------------
+=======================================================
```

Comment:
@ppiegaze @neverett for some reason docs generation was failing with `Title level inconsistent` even though https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections shows this is valid for a subsection - any suggestions?

Reply:
This is building locally for me without issue, are you still seeing a failure here?

Reply:
I see an issue if I leave it unchanged 😅

Reply:
Oh, I see why -- the "Configuring K8s Pods" heading needs to be enclosed in asterisks, since it's the first heading below the very top heading. If you're OK with me making changes to this branch, I can fix that in my edits.